.github/workflows/build.yml

```diff
@@ -28,14 +59,33 @@ jobs:
           name: JitDasm-netcoreapp3.0
           path: JitDasm/bin/Release/netcoreapp3.0
 
-  # Make sure it builds on Linux too
-  build-ubuntu:
-    name: Build (Ubuntu)
-    runs-on: ubuntu-latest
+      - name: Prepare .csrpoj for global-tool generation
+        shell: bash
+        run: sed -i "s#;net472</#</#" ./JitDasm/JitDasm.csproj
```

Comment:
It's better if you set an env var that the csproj can check and not include net472 if it's set (eg. with a Condition="...")

Reply:
Will give it a go (Not sure `<TargetFrameworks>` is that sort of a property)

Reply:
I already do something similar in dnlib: https://github.com/0xd4d/dnlib/blob/master/src/dnlib.csproj#L8-L12

Will dotnet pack fail if net472 is included?

Reply:
yes, net472 + pack == 💔 

I will try to take your `dnlib` thingy and see if it works. Looks promising!

Reply:
Works like a charm. Unsurprisingly :)